src/helm/benchmark/static/schema.yaml

```diff
@@ -335,7 +343,33 @@ models:
     num_parameters: 30000000000
     release_date: 2022-11-15
     todo: true
-
+  # Meta
+  - name: together/llama-7b
+    display_name: LLaMA (7B)
+    description: LLaMA is a collection of foundation language models ranging from 7B to 65B parameters.
+    creator_organization: Meta
+    model.access: open
+    num_parameters: 7000000000
+    release_date: 2023-02-24
+    todo: true
+  # Stanford
+  - name: together/alpaca-7b
+    display_name: Alpaca (7B)
+    description: Alpaca 7B is a model fine-tuned from the LLaMA 7B model on 52K instruction-following demonstrations
+    creator_organization: Stanford
+    model.access: open
+    num_parameters: 7000000000
+    release_date: 2023-03-13
+    todo: true
+  # LMSYS
+  - name: together/vicuna-13b
+    display_name: Vicuna (13B)
+    description: Vicuna-13B is an open-source chatbot trained by fine-tuning LLaMA on user-shared conversations collected from ShareGPT.
+    creator_organization: LMSYS
+    model.access: open
+    num_parameters: 13000000000
```

Comment:
Check sizes

Reply:
it's fine-tuned from LLaMA 13B which is 13.0B parameters according to the paper